server/routes/routeWorkers/standard.js

```diff
@@ -95,9 +73,54 @@ module.exports = ({ formConfig, licenceService, sectionName, nomisPushService, c
     res.redirect(`${nextPath}${bookingId}`)
   }
 
+  function getTarget(formName) {
+    const saveSection = formConfig[formName].saveSection || []
+    const targetSection = saveSection[0] || sectionName
```

Comment:
where you getting sectionName from?

Reply:
At creation. Line 4:

`module.exports = ({ formConfig, licenceService, sectionName, nomisPushService, config }) => {
  function get(req, res) {`